service/docs/source/geopmaccess.1.rst

```diff
@@ -1,28 +1,181 @@
-.. role:: raw-html-m2r(raw)
-   :format: html
 
-
-geopmaccess(1) -- Access management for the geopm service
+geopmaccess(1) -- Access management for the GEOPM Service
 =========================================================
 
+SYNOPSIS
+--------
+
+
+Read Access List
+~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess [-c] [-g GROUP | -a]
+
+
+Write Access List
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -w [-c] [-g GROUP] [-n | -s]
+
 
+Remove Access List
+~~~~~~~~~~~~~~~~~~
 
+.. code-block:: none
 
+    geopmaccess -D [-c] [-g GROUP] [-n]
+
+
+
+Get Help or Version
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -h | -v
 
 
 DESCRIPTION
 -----------
 
-Access management for the geopm service.
+The GEOPM service uses the ``/etc/geopm-service`` directory to store
+files that control which signals and controls may be accessed by a
+user through the service.  The purpose of the ``geopmaccess`` command
+line tool is to enable reading and writing of these access list files.
```

Comment:
One of the described use cases is for config validation. Maybe add a section or paragraph on failure scenarios.

I don't think it is necessary to enumerate every failure case (though I'm not against that if you want it). It should be sufficient to say what the user should expect to happen in pass/fail cases though. E.g., does it say that an error occurred, or what input caused the error? Does it set the process return code based on pass/fail state? How do options (e.g., dry-run or skip-check) impact this behavior?